src/init/caches.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_INIT_CACHES_H
+#define BITCOIN_INIT_CACHES_H
+
+#include <cstddef> // for size_t
+#include <cstdint> // for int64_t
+
+class ArgsManager;
+
+struct CacheSizes {
+    int64_t block_tree_db_cache_size;
```

Comment:
The `_cache_size` parts to all these member names seems redundant with the fact they're members of a struct called `CacheSizes`

Reply:
Fixed as of 6736e24a490d756e0e1e70b9e212775cf305ebe6